### PR TITLE
Updated GitTips.md 

### LINKED ISSUE #77: GitTips.md near 4 and 5
In comp 55, my team constantly had the issue when we pulled, and pushed a few, or one of us would not have a working project even when it was exactly the same as the working person's work. So another tip for if your project is not working after a pull try the following steps :
1. Gradle > Refresh Gradle
2. Build Path > Use as Source Folder
3. Commit and make sure all unstaged are dragged to staged changes. Commit and push

### PR DESCRIPTION
**Pull request by K5741**
Fixes #77

### CODE REVIEW

**Comment by onzfonz** [comment]
@K5741 need to use the `fixes` language specified in the assignment and fix the errors.  Please revisit https://pacific.instructure.com/courses/117102/assignments/558259 before I review it.

**Comment by K5741** [comment]
> @K5741 need to use the `fixes` language specified in the assignment and fix the errors. Please revisit https://pacific.instructure.com/courses/117102/assignments/558259 before I review it.

**Comment by onzfonz** [comment]
checks pass but the title should not have the fixes it should be in the description, please see #10 for clues on how to do this part.  Once this is there you can re-request review

**Comment by onzfonz** [comment]
@K5741 the branch needs to be updated and fixes still needs to be added to the description, not in the title, please see #10 before I can review

**Review by onzfonz — CHANGES_REQUESTED** [review]
please see comments about #10 that I left before I will review here), but also the brackets notation is not how we would do any commands, please use the arrow syntax `->` and use triple stars `***` to enclose any eclipse menu commands.

File: docs/labs/GitTips.md [hunk 1]
```diff
@@ -88,8 +88,19 @@ use you.
...
+Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
+to ensure that the project's dependencies are correctly synchronized.
+   - ***Build Path -> Use as Source Folder***
+This ensures that the pull is being recognized as source code by the IDE, allowing them to be
+compiled and executed correctly. This step helps in maintaining consistency between the working
+project and the changes you've pulled.
+   - ***Make sure all unstaged are dragged to staged changes when you commit and push***
+By dragging all unstaged changes to staged and then committing, you are ensuring that your local changes
+are ready to be pushed into the program. Pushing the commit then updates the
+remote repository with the new changes.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:102]:
```suggestion
  are ready to be pushed into the program.
  Pushing the commit then updates the remote repository with the new changes.
```

File: docs/labs/GitTips.md [hunk 2]
```diff
@@ -88,8 +88,19 @@ use you.
...
+5. If you are having project issues after a pull:
+   - ***Gradle -> Refresh Gradle***
+Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
+to ensure that the project's dependencies are correctly synchronized.
+   - ***Build Path -> Use as Source Folder***
+This ensures that the pull is being recognized as source code by the IDE, allowing them to be
+compiled and executed correctly. This step helps in maintaining consistency between the working
+project and the changes you've pulled.
+   - ***Make sure all unstaged are dragged to staged changes when you commit and push***
+By dragging all unstaged changes to staged and then committing, you are ensuring that your local changes
```
Comment (onzfonz) [inline docs/labs/GitTips.md:100]:
```suggestion
  By dragging all unstaged changes to staged and then committing, your local changes
```

File: docs/labs/GitTips.md [hunk 3]
```diff
@@ -88,8 +88,19 @@ use you.
...
-
-5. If you're having issues with the repository,
+5. If you are having project issues after a pull:
+   - ***Gradle -> Refresh Gradle***
+Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
+to ensure that the project's dependencies are correctly synchronized.
+   - ***Build Path -> Use as Source Folder***
+This ensures that the pull is being recognized as source code by the IDE, allowing them to be
+compiled and executed correctly. This step helps in maintaining consistency between the working
+project and the changes you've pulled.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:98]:
```suggestion
  This ensures that the pull is being recognized as source code by the IDE,
  allowing them to be compiled and executed correctly.
  This step helps in maintaining consistency between the working project
  and the changes you've pulled.
```

File: docs/labs/GitTips.md [hunk 4]
```diff
@@ -88,8 +88,19 @@ use you.
    your changes or the changes from the branch you're merging with.
    Typically, you'll select one set of changes but on occasion you may need to select changes from both.
    To get a better understanding of this, you can go to [step 8 of the GitHub Lab](9-Github.html#step-8-martyr--hackslash---commit-and-push-the-same-line).
-
-5. If you're having issues with the repository,
+5. If you are having project issues after a pull:
+   - ***Gradle -> Refresh Gradle***
+Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
+to ensure that the project's dependencies are correctly synchronized.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:94]:
```suggestion
   - ***Gradle -> Refresh Gradle***
  Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
  to ensure that the project's dependencies are correctly synchronized.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
please use the add suggestions to batch to get this to work, which must be done from the files changed tab on the pull request.

you'll do this from the files changes.  You still need to also edit the description

image

image

**Comment by onzfonz** [comment]
@K5741 you still need to update your branch and follow the add suggestions to batch information here

File: docs/labs/GitTips.md [hunk 5]
```diff
@@ -88,8 +88,20 @@ use you.
    your changes or the changes from the branch you're merging with.
    Typically, you'll select one set of changes but on occasion you may need to select changes from both.
    To get a better understanding of this, you can go to [step 8 of the GitHub Lab](9-Github.html#step-8-martyr--hackslash---commit-and-push-the-same-line).
-
-5. If you're having issues with the repository,
+5. If you are having project issues after a pull:
```
Comment (onzfonz) [inline docs/labs/GitTips.md:91]:
```suggestion
5. If you're having project issues after a pull, try:
```

File: docs/labs/GitTips.md [hunk 6]
```diff
@@ -88,8 +88,20 @@ use you.
...
+   - ***Gradle -> Refresh Gradle***
+ Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
+ to ensure that the project's dependencies are correctly synchronized.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:94]:
```suggestion
    Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
    to ensure that the project's dependencies are correctly synchronized.
```

File: docs/labs/GitTips.md [hunk 7]
```diff
@@ -88,8 +88,20 @@ use you.
...
+   - ***Build Path -> Use as Source Folder***
+ This ensures that the pull is being recognized as source code by the IDE,
+ allowing them to be compiled and executed correctly.
+ This step helps in maintaining consistency between the working project
+ and the changes you've pulled.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:99]:
```suggestion
    This ensures that the pull is being recognized as source code by the IDE,
    allowing them to be compiled and executed correctly.
    This step helps in maintaining consistency between the working project
    and the changes you've pulled.
```

File: docs/labs/GitTips.md [hunk 8]
```diff
@@ -88,8 +88,20 @@ use you.
...
+   - ***Make sure all unstaged are dragged to staged changes when you commit and push***
+ By dragging all unstaged changes to staged and then committing, your local changes
+ are ready to be pushed into the program.
+ Pushing the commit then updates the remote repository with the new changes.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:103]:
```suggestion
    By dragging all unstaged changes to staged and then committing, your local changes
    are ready to be pushed into the program.
    Pushing the commit then updates the remote repository with the new changes.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
let's try the request changes add to batch feature one more time @K5741

File: docs/labs/GitTips.md [hunk 9]
```diff
@@ -88,8 +88,20 @@ use you.
    your changes or the changes from the branch you're merging with.
    Typically, you'll select one set of changes but on occasion you may need to select changes from both.
    To get a better understanding of this, you can go to [step 8 of the GitHub Lab](9-Github.html#step-8-martyr--hackslash---commit-and-push-the-same-line).
-
-5. If you're having issues with the repository,
+5. If you're having project issues after a pull, try:
+   - ***Gradle -> Refresh Gradle***
+    Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
+    to ensure that the project's dependencies are correctly synchronized.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:94]:
```suggestion
     Sometimes, after pulling changes from the program, the Gradle configuration might need to be refreshed
     to ensure that the project's dependencies are correctly synchronized.
```

File: docs/labs/GitTips.md [hunk 10]
```diff
@@ -88,8 +88,20 @@ use you.
...
+   - ***Build Path -> Use as Source Folder***
+    This ensures that the pull is being recognized as source code by the IDE,
+    allowing them to be compiled and executed correctly.
+    This step helps in maintaining consistency between the working project
+    and the changes you've pulled.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:99]:
```suggestion
     This ensures that the pull is being recognized as source code by the IDE,
     allowing them to be compiled and executed correctly.
     This step helps in maintaining consistency between the working project
     and the changes you've pulled.
```

File: docs/labs/GitTips.md [hunk 11]
```diff
@@ -88,8 +88,20 @@ use you.
...
+   - ***Make sure all unstaged are dragged to staged changes when you commit and push***
+    By dragging all unstaged changes to staged and then committing, your local changes
+    are ready to be pushed into the program.
+    Pushing the commit then updates the remote repository with the new changes.
```
Comment (onzfonz) [inline docs/labs/GitTips.md:103]:
```suggestion
     By dragging all unstaged changes to staged and then committing, your local changes
     are ready to be pushed into the program.
     Pushing the commit then updates the remote repository with the new changes.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
do not commit unless you see the add suggestions to batch button enabled in the files changed tab and then after clicking all three you click the commit suggestions button (just once)
Please verify with a screenshot

**Review by onzfonz — APPROVED** [review]
LGTM

**Comment by onzfonz** [comment]
Now all you have to do is update the branch and then you'll be able to merge

**Pull request merged**